cg/apps/gt.py

```diff
@@ -37,3 +47,33 @@ def upload(self, bcf_path: str, samples_sex: dict, force: bool=False):
                 analysis_obj.sex = samples_sex[analysis_obj.sample_id]['analysis']
                 analysis_obj.sample.sex = samples_sex[analysis_obj.sample_id]['pedigree']
                 self.commit()
+
+
+    def get_trending(self, sample_id : str = '' , days: str = '' ):
+        """Get trending object with one or many samples."""
+        trending_obj = None
+        trending_call = copy.deepcopy(self.base_call)
+        
+        if sample_id:
+            trending_call.extend(['prepare-trending', '-s', sample_id])
+        elif days:
+            trending_call.extend(['prepare-trending', '-d', days])
+
+        try:
+            output = subprocess.check_output(
+                ' '.join(trending_call),
+                shell=True
+            )
+        except CalledProcessError:
+            # If CalledProcessError is raised, log and raise error
+            log_msg = f"Could not run command: {' '.join(trending_call)}"
+            LOG.critical(log_msg)
+            raise
```

Comment:
Please raise the original exception as well.

There is a good explanation on why to do so in "Clean Code for Python", page 60 and onward. Raising original exception is on page 68.

Reply:
👍

Reply:
I dont get what is the original exception here.. @ingkebil

Reply:
It's the `CalledProcessError`